torch/export/_trace.py

```diff
@@ -1288,29 +1280,29 @@ def forward(self, *args, **kwargs):
 
     fake_params_buffers = make_fake_params_buffers(fake_mode, _get_params_buffers(mod))
 
-    with fake_mode:
-        with _fakify_script_objects(mod, fake_args, fake_kwargs, fake_mode) as (
+    with _fakify_script_objects(mod, fake_args, fake_kwargs, fake_mode) as (
+        patched_mod,
+        new_fake_args,
+        new_fake_kwargs,
+        new_fake_constant_attrs,
+        map_fake_to_real,
+    ):
+        aten_export_artifact = _export_to_aten_ir(
             patched_mod,
             new_fake_args,
             new_fake_kwargs,
+            fake_params_buffers,
+            fake_mode,
```

Comment:
Instead of adding fake_mode here, why not just add fake_mode as context manager at the top of this function call?